.github/workflows/autotools-cmake.yml

```diff
@@ -128,8 +127,9 @@ jobs:
           # Autotools' LT_LIB_M has a hardcoded exclude for "*-*-darwin*" hosts,
           # while macOS does have libm and is successfully found by CMake.
           # We patch the CMake side in line here to get the differ below to empty.
-          export PATH="$(brew --prefix)/opt/gnu-sed/libexec/gnubin:${PATH}"
-          sed 's,-lm,,' -i build_cmake/ROOT/usr/local/lib*/pkgconfig/expat.pc
+          sed -e 's,-lm,,' -i.bak \
+              build_cmake/ROOT/usr/local/lib*/pkgconfig/expat.pc
+          rm -f expat.pc.bak
```

Comment:
```diff
--- rm -f expat.pc.bak
+++ rm -f build_cmake/ROOT/usr/local/lib*/pkgconfig/expat.pc.bak
```
?

Reply:
yeah I should clearly not be trying to push quick fixes at midnight on a Friday